app/en/home/build-tools/create-a-mcp-server/page.mdx

```diff
@@ -104,25 +104,43 @@ if __name__ == "__main__":
 Run your MCP Server using one of the with the following commands in your terminal:
 
 <Tabs
-  items={["HTTP transport (default)", "stdio transport (for Claude Desktop)"]}
+  items={["stdio transport (default)", "http transport"]}
   storageKey="preferredTransport"
 >
 
 <Tabs.Tab>
 
 ```bash
-uv run server.py http
+uv run server.py stdio
 ```
 
-For HTTP transport, view your server's API docs at [http://127.0.0.1:8000/docs](http://127.0.0.1:8000/docs).
+<Callout type="info">
```

Comment:
could call out that `arcade configure` will add those secrets to the client's config

Reply:
I agree, there's [this PR](https://github.com/ArcadeAI/docs/pull/516) that talks about configuring MCP clients. I have a TODO to link to that from here when merged.